lte/gateway/python/magma/pipelined/rule_mappers.py

```diff
@@ -146,7 +146,8 @@ def get_version(self, imsi: str, ip_addr: IPAddress, rule_id: str) -> int:
         with self._lock:
             version = self._version_by_imsi_and_rule.get(key)
             if version is None:
-                version = -1
+                """Set to zero to allow proper cleanup of old flows"""
```

Comment:
throws ```W0105: 149, 16: String statement has no effect (pointless-string-statement)```
replace with ```#Set to zero to allow proper cleanup of old flows```

Reply:
@koolzz fixed. actually, i followed the same style as the other comments in the file.

Reply:
linter works in mysterious ways, I think its just due to it being on one line. Thanks!